proguard/src/main/java/com/flurry/proguard/AndroidUploadType.java

```diff
@@ -0,0 +1,24 @@
+package com.flurry.proguard;
+
+public enum AndroidUploadType {
+    ANDROID_JAVA("ANDROID", "ProGuard"),
+    ANDROID_NATIVE("ANDROID_NATIVE", "Native")
```

Comment:
Missing trailing comma here?

Reply:
Not sure, hy should there be a trailing a comma here?

Reply:
cleaner git diffs